js-pkg/create-y-sweet-app/src/templates/nextjs/.env.local

```diff
@@ -0,0 +1 @@
+Y_SWEET_CONNECTION_STRING="ys://127.0.0.1:8080"
```

Comment:
Thoughts on changing this to `CONNECTION_STRING` so it matches the instructions the `y-sweet serve` command gives for running with the examples? Might help reduce confusion by being consistent across those.

<img width="795" alt="Screenshot 2024-11-21 at 6 26 58 PM" src="https://github.com/user-attachments/assets/5000b2e7-177a-4adc-8fee-eff76002309b">